src/layouts/Banner/index.jsx

```diff
@@ -0,0 +1,51 @@
+import aImage from "../../assets/images/a.jpg";
+import bImage from "../../assets/images/b.jpg";
+
+import cImage from "../../assets/images/c.jpg";
+
+import "./style.css";
+
+const bands = () => {
```

Comment:
ani yo component ko naam bands kina cha?
Components name should always be capital